src/services/ProjectService.ts

```diff
@@ -164,20 +166,46 @@ export class ProjectService {
   }
 
   private static async createProject(proposal: ProposalWithOutcome) {
-    const newProject: ProjectAttributes = {
+    const creationDate = new Date()
+    const newProject = await ProjectModel.create({
       id: crypto.randomUUID(),
       proposal_id: proposal.id,
       title: proposal.title,
       status: ProjectStatus.Pending,
       links: [],
-      created_at: new Date(),
-    }
+      created_at: creationDate,
+    })
+
+    await ProjectService.createPersonnel(proposal, newProject, creationDate)
 
-    return await ProjectModel.create(newProject)
+    return newProject
+  }
+
+  private static async createPersonnel(
+    proposal: ProposalWithOutcome,
+    newProject: ProjectAttributes,
```

Comment:
This won't be always a new project, isn't it? Maybe we can call it `project` here.

Reply:
Hmm I agree on the name change, but in this case we will always use a new project, since the `createProject` method is the only one calling this, and it uses the personnel from an existing proposal to create all the personnel for a new project.

If we want to create new personnel for a project that already exists, we should have another method that creates them one by one (this one uses `createMany`)

WDYT?